crates/graphql/src/schema/objects/bid_receipt.rs

```diff
@@ -88,6 +84,14 @@ impl BidReceipt {
             .await
             .map_err(Into::into)
     }
+
+    pub async fn auction_house(&self, context: &AppContext) -> FieldResult<Option<AuctionHouse>> {
+        context
+            .auction_house_loader
+            .load(self.auction_house.to_owned().into())
```

Comment:
What does the auction_house_loader use ideally it should take `PublicKey<AuctionHouse>`

Reply:
We can have 2 data loaders 1 that takes auction house address and looks up auction_house and another store_config_auction_houses that accepts PublicKey<StoreConfig> and returns list of auction house based on the config address.